docs/management/persistence.md

```diff
@@ -259,26 +259,28 @@ and starts appending new data into the new file.
 
 ### How I can switch to AOF, if I'm currently using dump.rdb snapshots?
 
-There is a different procedure to do this in version 2.0 and later versions, as you
-can guess it's simpler since Redis 2.2 and does not require a restart at all.
+If you want to enable AOF in a server that is currently using RDB snapshots, you need to convert the data by enabling AOF via CONFIG command on the live server first.
+
+**IMPORTANT:** not following this procedure (e.g. just changing the config and restarting the server) can result in data loss!
 
 **Redis >= 2.2**
 
 * Make a backup of your latest dump.rdb file.
 * Transfer this backup to a safe place.
-* Issue the following two commands:
-* `redis-cli config set appendonly yes`
-* `redis-cli config set save ""`
-* Make sure your database contains the same number of keys it contained.
+* Enable AOF: `redis-cli config set appendonly yes`
+* Optionally disable RDB: `redis-cli config set save ""`
+* Make sure your database contains the same number of keys it contained previously.
 * Make sure writes are appended to the append only file correctly.
+* Update your `redis.conf` and ensure that it matches the config above.
```

Comment:
maybe add a reference to CONFIG REWRITE?

Reply:
Maybe like this?

> * Update your `redis.conf` (potentially through `CONFIG REWRITE`) and ensure that it matches the config above.